Makefile

```diff
@@ -22,4 +22,5 @@ release:
 
 .PHONY: clean
 clean:
-	rm -rf dist build *.egg-info .tox
+	rm -rf dist build *.egg-info .tox .venv
+	find . -type d -name '__pycache__' -exec rm -r {} +
```

Comment:
I think I've seen this as `\+` at the end of the find exec, is there a difference?

Reply:
Ahh, looks like the semicolon version _does_ need escaping, while the plus one does not (but people write it that way anyways).

Reply:
A very subtle one. If `+` is a special character in your shell, `\+` would ensure the literal `+` makes it to `find`.

I don't think this is an issue in make, since the default shell is `/bin/sh`.
We could tweak it if you like, but I've already merged because the cryptography thing is in the way of getting everything else merge-ready.

Reply:
Yep, totally fine. Just a curiosity.

Reply:
> Just a curiosity.

Like the rest of the syntax of `find -exec`! 😹 

You'll also sometimes see single quotes, for similar reasons. It gets... very confusing. But it's the "easiest" way I know of to clean up files and dirs in a tree.

The change made it in here because I was tinkering a lot with the warnings stuff and thought maybe a pycache dir was causing issues. (It was not.)